test/functional/mempool_truc.py

```diff
@@ -163,30 +159,6 @@ def test_truc_replacement(self):
         self.check_mempool([tx_v3_bip125_rbf_v2["txid"], tx_v3_parent["txid"], tx_v3_child["txid"]])
 
 
-    @cleanup(extra_args=["-mempoolfullrbf=0"])
-    def test_truc_bip125(self):
-        node = self.nodes[0]
-        self.log.info("Test TRUC transactions that don't signal BIP125 are replaceable")
-        assert_equal(node.getmempoolinfo()["fullrbf"], False)
```

Comment:
It seems to me that we would still want to test that TRUC transactions that don’t signal replaceability can be replaced. Obviously the assert_equal here that checks that the node does not use fullrbf would fail, but the rest of the test seems like it could be kept.